packages/react-core/src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -59,7 +59,7 @@ export const BreadcrumbItem: React.FunctionComponent<BreadcrumbItemProps> = ({
       {isDropdown && <span className={css(styles.breadcrumbDropdown)}>{children}</span>}
       {render && render({ className, ariaCurrent })}
       {to && !render && (
-        <Component href={to} target={target} className={className} aria-current={ariaCurrent}>
+        <Component href={to} target={target} className={className} aria-current={ariaCurrent} {...props}>
```

Comment:
This will spread `props` over two places (here and the `li` above). Adding another property with `any` type such as `componentProps` and spreading that may make more sense than overloading `props`.

Reply:
Good catch! I actually didn't notice these props were already being spread over the root element, which arguably makes a lot more sense.